ex1.py

```diff
@@ -0,0 +1,11 @@
+def my_filter(function,iterable):
+    """ run on the iterable and execute the function , we add the element only if the retrun value of 
+    the function is not false or 0"""
```

Comment:
Write documentation in the right format -
python
def count_hi_v4(word: str) -> int:
"""
The function counts how many times the string "hi" appears in a given string.
@param word: The given string.
@return: The amount of times the word "hi" appeared.
"""
return word.count("hi")